src/main/java/uk/gov/justice/digital/delius/data/api/CustodialStatus.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.justice.digital.delius.data.api;
+
+import lombok.Builder;
+import lombok.Getter;
+
+import java.time.LocalDate;
+
+@Getter
+@Builder
+public class CustodialStatus {
```

Comment:
So this is a sort of DTO, that takes stuff from the event , the disposal and custody, which is of course fine.
I suppose we could have returned a wrapper object that contained

- uk.gov.justice.digital.delius.data.api.Sentence
- uk.gov.justice.digital.delius.data.api.Custody 

some sort of CustodialSentence class. There is a danger we end up with a ton of different variations for each minor use case. Since we have duplicated the dates that were already in CustodyRelatedKeyDates for instance.

The thing missing if we did this is `offence` which is already in the conviction, those that indicate we are returning information the client has already from the /convictions endpoint?

Have a think if you think it is worth generalising this or if it is more trouble then it is worth. I'm not sure either way, just slightly worried that we might end up with two many endpoints that are too specific to very specific UIs